src/CodeCoverageExtension.php

```diff
@@ -118,7 +118,7 @@ public function load(ServiceContainer $container, array $params = [])
 
             $skipCoverage = false;
             $input = $container->get('console.input');
-            if ($input->getOption('no-coverage')) {
+            if (!$input->hasOption('no-coverage') || $input->getOption('no-coverage')) {
```

Comment:
I think it would be more readable to have it check if it has an option AND if it's enabled instead:

```php
if ($input->hasOption('no-coverage') && $input->getOption('no-coverage')) {
```

Functionally it is the same as the original, but I believe it is a bit easier to read.

Reply:
That's not exactly the same, because it won't skip coverage if the option is missing.

Reply:
(in my case, I don't want coverage to be executed when creating a spec!)

Reply:
I understand, for this we need to add `--no-coverage` option to the remaining command (currently only `run` has that option). I will sort this out shortly and issue a bugfix release.

Reply:
Thanks!